src/monumenten/_processing.py

```diff
@@ -50,6 +53,11 @@ async def _process_batch(
         rijksmonumenten_taak, verblijfsobjecten_taak
     )
 
+    if not verblijfsobjecten:
+        raise ValueError(
+            "Geen geldige BAG verblijfsobjecten gevonden voor de opgegeven IDs"
```

Comment:
```suggestion
            "Geen geldige BAG verblijfsobjecten gevonden voor een complete batch"
```
@benverhees misschien is dit beter, want het gaat op batch-niveau